src/components/Assets/FilterAssetComponent.js

```diff
@@ -12,73 +13,83 @@ const placeCategoriesInSemanticUIOptions = props => props.map((option, index) =>
   value: option.category_name
 }));
 
-const FilterAssetComponent = props => (
-  <div className="modal-container" >
-    <div className="page-indicator">
-      <div className={props.page === 0 ? 'circle shade-2' : 'circle shade-1'}>1</div>
-      Identify your device
-      <div className="line" />
-      <div className="circle">2</div>
-      Fill out device specs
+const FilterAssetComponent = (props) => {
+  if (Object.values(props.isLoadingState).find(loading => loading === true)) {
+    return (
+      <div className="modal-container" >
+        <LoaderComponent />
+      </div>
+    );
+  }
+
+  return (
+    <div className="modal-container" >
+      <div className="page-indicator">
+        <div className={props.page === 0 ? 'circle shade-2' : 'circle shade-1'}>1</div>
```

Comment:
`0` is a falsy value. Therefore, this can be simplified to:
```
props.page ?  'circle shade-1' : 'circle shade-2'
```
Please confirm if the change renders the correct circle and color